iOS/UatsApp/LoginVC.swift

```diff
@@ -0,0 +1,180 @@
+//        let isLoggedin:Int = prefs.integerForKey("ISLOGGEDIN") as Int
+//        
+//        if(isLoggedin == 1){
+//            self.performSegueWithIdentifier("goApp", sender: self)
+//        }
+    }
+    
+    
+    @IBOutlet weak var signin: UIButton!
+    
+    @IBAction func signinTapped(sender: UIButton) {
+        var username:NSString = txtUsername.text
+        var password:NSString = txtPassword.text
+        
+        if ( username.isEqualToString("") || password.isEqualToString("") ) {
+            
+            var alertView:UIAlertView = UIAlertView()
+            alertView.title = "Sign in Failed!"
+            alertView.message = "Please enter Username and Password"
+            alertView.delegate = self
+            alertView.addButtonWithTitle("OK")
+            alertView.show()
+        } else {
+            
+            var post:NSString = "username=\(username)&password=\(password)"
+            
+            NSLog("PostData: %@",post);
+            
+           // var url:NSURL = NSURL(string: "http://uatsapp.16mb.com/register/jsonlogin2.php")!
+            var url:NSURL = NSURL(string: "http://uatsapp.tk/registerDEV/jsonlogin1.php")!
+            
+            
+            var postData:NSData = post.dataUsingEncoding(NSASCIIStringEncoding)!
+            
+            var postLength:NSString = String( postData.length )
+            
+            var request:NSMutableURLRequest = NSMutableURLRequest(URL: url)
+            request.HTTPMethod = "POST"
+            request.HTTPBody = postData
+            request.setValue(postLength as String, forHTTPHeaderField: "Content-Length")
+            request.setValue("application/x-www-form-urlencoded", forHTTPHeaderField: "Content-Type")
+            request.setValue("application/json", forHTTPHeaderField: "Accept")
+            
+            
+            var reponseError: NSError?
+            var response: NSURLResponse?
+            
+            var urlData: NSData? = NSURLConnection.sendSynchronousRequest(request, returningResponse:&response, error:&reponseError)
+            
+            if ( urlData != nil ) {
+                let res = response as! NSHTTPURLResponse!;
+                
+                NSLog("Response code: %ld", res.statusCode);
+                
+                if (res.statusCode >= 200 && res.statusCode < 300)
+                {
+                    var responseData:NSString  = NSString(data:urlData!, encoding:NSUTF8StringEncoding)!
+                    
+                    NSLog("Response ==> %@", responseData);
+                    
+                    var error: NSError?
+                    
+                    let jsonData:NSDictionary = NSJSONSerialization.JSONObjectWithData(urlData!, options:NSJSONReadingOptions.MutableContainers , error: &error) as! NSDictionary
+                    
+                    
+                    let success:NSInteger = jsonData.valueForKey("success") as! NSInteger
```

Comment:
Try to convert to boolean if the value is true/false.. even json supports it as a primitive type..

always name your booleans starting with: is, has, will, can